tests/unit/test_load.py

```diff
@@ -20,6 +21,20 @@ def test_encode_chunk_with_unicode():
     assert u'信用卡' in csv_string
 
 
+def test_encode_chunk_with_floats():
+    """Test that floats in a dataframe are encoded with at most 15 significant
+        figures.
+
+    See: https://github.com/pydata/pandas-gbq/issues/192
+    """
+    input_csv = StringIO('01/01/17 23:00,1.05148,1.05153,1.05148,1.05153,4')
```

Comment:
For Py2 to pass, you need to put `StringIO(u'...')` to pass unicode into `StringIO` (you can see the test failure in Travis)